lib/ex_plasma/output/type/generic_payment.ex

```diff
@@ -0,0 +1,107 @@
+defmodule ExPlasma.Output.Type.GenericPayment do
+  @moduledoc """
+  Payment V1 Output Type.
+  """
+
+  @behaviour ExPlasma.Output
+
+  alias ExPlasma.Output
+
+  @type address() :: <<_::160>>
+  @type output_guard() :: address()
+  @type token() :: address()
+  @type amount() :: non_neg_integer()
+
+  @type rlp() :: [output_guard() | [token() | amount()]]
+
+  @type validation_responses() :: {:ok, t()}
+
+  @type t() :: %{
+          output_guard: output_guard(),
+          token: token(),
+          amount: amount()
+        }
+
+  @zero_address <<0::160>>
+
+  @doc """
+  Encode a map of the output data into an RLP list.
+
+  ## Example
+
+  iex> output = %{output_type: 1, output_data: %{output_guard: <<1::160>>, token: <<1::160>>, amount: 1}}
+  iex> ExPlasma.Output.Type.PaymentV1.to_rlp(output)
```

Comment:
shouldn't `ExPlasma.Output.Type.GenericPayment` be used here and below?

Reply:
Nice catch, forgot this one when moving things around!